fields/types/relationship/RelationshipField.js

```diff
@@ -218,7 +218,7 @@ module.exports = Field.create({
 					{this.renderSelect()}
 				</Section>
 				<Section>
-					<Button onClick={this.openCreate} type="success">+</Button>
+					<Button onClick={this.openCreate} type="button">+</Button>
```

Comment:
we can probably remove the `type="button"` entirely, given https://github.com/keystonejs/keystone/blob/master/admin/client/App/elemental/Button/index.js#L57